### PR TITLE
feat(docs): add documentation

### PR DESCRIPTION
**Pull request by jzeuzs**
Adds html-generated documentation. (as requested by @fuwwy on Discord)

Preview:

### CODE REVIEW

**Comment by Isaiah-Hamilton** [comment]
@1chiSensei  Are you using plain HTML or a Javascript framework like React.js or Angular.

**Comment by jzeuzs** [comment]
> @1chiSensei  Are you using plain HTML or a Javascript framework like React.js or Angular.

Plain HTML. So it can be served with github pages.

**Comment by Isaiah-Hamilton** [comment]
@1chiSensei  I have some changes I want to make to your code but I don't know how to commit to your pull request. Can you help me with that? Edit: I found a workaround for the moment

File: docs/assets/bass.css [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+@media (min-width: 52em) and (max-width: 64em){
+  .md-hide{ display:none !important }
+}
+
+@media (min-width: 64em){
+  .lg-hide{ display:none !important }
+}
+
+.display-none{ display:none !important }
+
```
Comment (Isaiah-Hamilton) [inline docs/assets/bass.css:544]:
```suggestion

.page_github {
  float: left;
  padding-right: 7px;
  padding-top: 1px;
  color: #fff;
}

.page__subtitle {
  float: left;
}

```

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,8413 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8' />
+  <title>moonstone-wrapper 0.0.6 | Documentation</title>
+  <meta name='viewport' content='width=device-width,initial-scale=1'>
+  <link href='assets/bass.css' rel='stylesheet' />
+  <link href='assets/style.css' rel='stylesheet' />
+  <link href='assets/github.css' rel='stylesheet' />
+  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,700i' />
```
Comment (Isaiah-Hamilton) [inline docs/index.html:10]:
```suggestion
  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,700i' />
  <script src="https://kit.fontawesome.com/a223b19e18.js" crossorigin="anonymous"></script>
```

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,8413 @@
...
+</head>
+<body class='documentation m0'>
+  <div class='page flex'>
+    <header class='page__header'>
+      <div class='inner'>
+        <div class='page__headline'>
+          <h1 class='mb0 no-anchor page__title'>moonstone-wrapper</h1>
```
Comment (Isaiah-Hamilton) [inline docs/index.html:17]:
```suggestion
          <h1 class='mb0 no-anchor page__title'>moonstone-wrapper</h1>
          <a href="https://github.com/fuwwy/moonstone" class="page_github"><i class="fab fa-github"></i></a>
```

**Comment by Isaiah-Hamilton** [comment]
This was the only way that I found that I could add code. This code adds a GitHub button in front of the npm package number that takes you to the GitHub repo.

**Comment by Isaiah-Hamilton** [comment]
@fuwwy Why was this pull request closed. edit nvm

**Pull request closed without merging**